deploy/kubernetes/elastic-agent-standalone-kubernetes.yml

```diff
@@ -87,9 +109,120 @@ data:
       node: ${NODE_NAME}
       scope: node
     inputs:
-      - id: 934ef8aa-ed19-405b-8160-ebf62e3d32f8
-        name: kubernetes-node-metrics
-        revision: 1
+      - name: log-1
+        type: logfile
+        use_output: default
+        meta:
+          package:
+            name: log
+            version: 0.4.6
+        data_stream:
+          namespace: default
+        streams:
+          - data_stream:
+              dataset: generic
+            symlinks: true
+            paths:
+              - /var/log/containers/*${kubernetes.container.id}.log
+      - name: system-3
```

Comment:
Out of curiosity, how did you generate this config? In Kibana UI?

Reply:
I used Kibana's Fleet UI to generate it. This how we propose users to do in standalone version (https://www.elastic.co/guide/en/fleet/current/run-elastic-agent-standalone.html).